app/src/main/res/layout/ss_about_activity_v2.xml

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can we replace the old `ss_about_activity.xml` instead of creating a `v2` layout